src/styles/utils/variables.scss

```
@@ -0,0 +1,11 @@
+$flex__gap: 1px;
+$day__size: 100px;
+
+$days:
+  "mon" 0,
+  "tue" $day__size * 1 + $flex__gap * 1,
+  "wed" $day__size * 2 + $flex__gap * 2,
```

You can only calculate your margin once, instead of calculating it every time
$day__size + $flex__gap
```suggestion
  "tue" $dayMargin;
  "wed" 2 * $dayMargin,
```

---

totally agree